terraform/SLI_3_days_views.tf

```diff
@@ -71,7 +71,7 @@ resource "google_bigquery_table" "SLI_3_days_view" {
             census.datasetId=last_backups.source_dataset_id AND
             census.tableId=last_backups.source_table_id
           WHERE
-            IFNULL(census.partitionId, 'null')=IFNULL(last_backups.source_partition_id, 'null')
+            (census.partitionId=last_backups.source_partition_id OR last_backups.source_partition_id IS NULL)
```

Comment:
I would invert the equal statement so that it's easier to read, i.e.:
`(last_backups.source_partition_id=census.partitionId OR last_backups.source_partition_id IS NULL)`